File: src/main/java/hr/fer/nmbp/text_search/model/Analysis.java
package hr.fer.nmbp.text_search.model;

import lombok.Data;

import java.util.List;

@Data
public class Analysis {

    private List<String> columns;
    private List<Object[]> rows;

    public Analysis(List<String> columns, List<Object[]> rows) {
        this.columns = columns;
        this.rows = rows;
    }
}

File: frontend/src/app/Analysis.ts
export class Analysis {

  private _columns: string[];
  private _rows: Object[][];


  constructor(dates: string[], rows: Object[][]) {
    this._columns = dates;
    this._rows = rows;
  }


  get columns(): string[] {
    return this._columns;
  }

  get rows(): Object[][] {
    return this._rows;
  }
}
